gatsby-config.js

```diff
@@ -1,6 +1,6 @@
 require('dotenv').config();
 
-const config = require('./src/config');
+const config = require('./src/config.json');
```

Comment:
Do we need to add the extension here?

Reply:
yes because I changed the config to json. I guess its not needed anymore, but since that file doesnt do anything else might as well make it `json` in case we ever need to update it through an action in the future